frontend/packages/console-app/src/components/cluster-configuration/useClusterConfigurationGroupItems.ts

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
Can you rename this filename `useClusterConfigurationGroupItems.ts` as it does not have any hook?

Reply:
Changed it to `getClusterConfigurationGroups`